### DISCUSSION TITLE
5901E102-27C7-40B6-BF7D-97C6F6DAB6CE

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#130] [General]
# 5901E102-27C7-40B6-BF7D-97C6F6DAB6CE

HD 19916, also known as TOI-257, is an F-type subgiant star about 251.38 light-years away in the constellation Horologium. A single gas giant planet about 2/3 the size of Saturn completes a single orbit once every 13.55 days. A larger planet was speculated to exist, but it wasn't confirmed yet by follow-up studies

https://celestia.mobi/addon?item=5901E102-27C7-40B6-BF7D-97C6F6DAB6CE

**Comment by exocritic5457** [comment]
I would definitely feel a certain way if I was a recolored, low-resolution version of the upper part of Jupiter, and it wouldn't be a good feeling at that
